### ISSUE TITLE
PEP 8 check

### ISSUE THREAD

**Issue by lunemec** [jonathanslenders/pyvim#29]
I'm not exactly sure how the pyflakes works with files, but I'm missing the ability to see pep8 warnings and other in small horizontal split below the open file. This is extremely handy feature.

**Comment by jonathanslenders** [comment]
Yes, that's something I'd like to have as well. Probably I will do it somewhere in the near future.

**Comment by lunemec** [comment]
This seems to be related with #47
